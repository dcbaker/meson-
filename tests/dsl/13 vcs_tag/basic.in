This is a @VCS_TAG@ and @VCS_TAG@
    @VCS_TAG@ @@VCS_TAG@ @VCS_TAG
