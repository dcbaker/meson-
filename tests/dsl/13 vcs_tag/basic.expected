This is a 1.0.0 and 1.0.0
    1.0.0 @1.0.0 @VCS_TAG
